hr_employee_birth_name/readme/INSTALL.rst

```diff
@@ -0,0 +1,8 @@
+To install this module, you need to:
+
+1.  Clone the branch 11.0 of the repository https://github.com/OCA/hr
```

Comment:
11.0 should change now to 16.0
Anyway for me these instructions seems like not needed at all, it's too basic